README.md

```diff
@@ -55,21 +60,21 @@ More documentation can be found in the [forge package](./forge/README.md) and in
 
 ### Features
 
-1. Fast & flexible compilation pipeline:
-    1. Automatic Solidity compiler version detection & installation (under
+- **Fast & flexible compilation pipeline**
+    - Automatic Solidity compiler version detection & installation (under
        `~/.svm`)
-    1. Incremental compilation & caching: Only changed files are re-compiled
-    1. Parallel compilation
-    1. Non-standard directory structures support (e.g. can build
+    - **Incremental compilation & caching**: Only changed files are re-compiled
+    - Parallel compilation
+    - Non-standard directory structures support (e.g. 
        [Hardhat repos](https://twitter.com/gakonst/status/1461289225337421829))
-1. Tests are written in Solidity (like in DappTools)
-1. Fast fuzz Tests with shrinking of inputs & printing of counter-examples
-1. Fast remote RPC forking mode leveraging Rust's async infrastructure like tokio
-1. Flexible debug logging:
-    1. Dapptools-style, using `DsTest`'s emitted logs
-    1. Hardhat-style, using the popular `console.sol` contract
-1. Portable (5-10MB) & easy to install statically linked binary without requiring Nix or any other package manager
-1. Abstracted over EVM implementations (currently supported: Sputnik, EvmOdin)
+- **Tests are written in Solidity** (like in DappTools)
+- **Fast fuzz testing** with shrinking of inputs & printing of counter-examples
+- **Fast remote RPC forking mode**, leveraging Rust's async infrastructure like tokio
+- **Flexible debug logging**
+    - Dapptools-style, using `DsTest`'s emitted logs
+    - Hardhat-style, using the popular `console.sol` contract
+- **Portable (5-10MB) & easy to install** without requiring Nix or any other package manager
+- **Abstracted over EVM implementations** (currently supported: Sputnik, EvmOdin)
```

Comment:
Should we still list this? AFAIK this is no longer the case

Reply:
+1 on removing this, perhaps we can highlight the github action instead?

Reply:
Done!